packages/grafana-toolkit/src/config/webpack.plugin.config.ts

```diff
@@ -224,3 +228,18 @@ export const getWebpackConfig: WebpackConfigurationGetter = options => {
     optimization,
   };
 };
+
+export const loadWebpackConfig: WebpackConfigurationGetter = options => {
+  const baseConfig = getBaseWebpackConfig(options);
+
+  const customWebpackPath = path.resolve(process.cwd(), 'webpack.plugin.config.ts');
```

Comment:
Why not naming this file webpack.config.ts?

Reply:
I wanted to differ it from the default naming of webpack.
If we use the default webpack.config.ts everybody expects a custom config as proposed by steven.